docs/providers/overview.mdx

```diff
@@ -268,6 +268,14 @@ By leveraging Keep Providers, users are able to deeply integrate Keep with the t
   }
 ></Card>
 
+<Card
+  title="Graylog"
+  href="/providers/documentation/graylog-provider"
+  icon={
+    <img src="https://img.logo.dev/graylog.com?token=pk_dfXfZBoKQMGDTIgqu7LvYg" />
```

Comment:
please use the icon on public instead of using third party with token that might break